Src/AutoFixture.xUnit.net2/CompositeDataAttribute.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Reflection;
+using Xunit.Sdk;
+
+namespace Ploeh.AutoFixture.Xunit2
+{
+    /// <summary>
+    /// An implementation of DataAttribute that composes other DataAttribute instances.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Method, AllowMultiple = true, Inherited = true)]
+    [CLSCompliant(false)]
+    [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Performance", "CA1813:AvoidUnsealedAttributes", Justification = "This attribute is the root of a potential attribute hierarchy.")]
+    public class CompositeDataAttribute : DataAttribute
+    {
+        private readonly IEnumerable<DataAttribute> attributes;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="CompositeDataAttribute"/> class.
+        /// </summary>
+        /// <param name="attributes">The attributes representing a data source for a data theory.
+        /// </param>
+        public CompositeDataAttribute(IEnumerable<DataAttribute> attributes)
```

Comment:
IMO, the whole point of this is to apply it *at least* at the constructor parameter (because that's where the (potential bad) input may come from). :smiley: